test-clients/src/main/java/com/hedera/services/bdd/suites/token/OCTokenSpec.java

```diff
@@ -1,4 +1,4 @@
-package com.hedera.services.bdd.suites.contract;
+package com.hedera.services.bdd.suites.token;
```

Comment:
The name suggests that this test is for tokens, but actually it tests the HCS service. It should be in the HCS e2e tests

Reply:
Removed OCTokenSpec.java and moved the only test the file contained to consensus.SubmitMessageSuite.java